### ISSUE TITLE
Problems on data import with PG_SUPPORTS_IDENTITY activated

### ISSUE THREAD

**Issue by R0Wi** [darold/ora2pg#862]
Hi and thank you for this nice tool :-)

I was playing around a little bit with latest version 20, trying to migrate some data from a Oracle 18 database to PostgreSQL 12. After the dump i had the `TABLE_output.sql`, `INSERT_output.sql` and `AUTOINCREMENT_output.sql` files generated properly. The `TABLE`-file contains some tables with `ID`-columns, generated correctly as `IDENTITY` (because the `PG_SUPPORTS_IDENTITY` is activated by default) like
```
CREATE TABLE mytable (
id bigint GENERATED ALWAYS AS IDENTITY ...
);
```
Now when trying to execute the `INSERT`-file Postgres gives me the following error:
```
Err] ERROR:  cannot insert into column "id"
DETAIL:  Column "id" is an identity column defined as GENERATED ALWAYS.
HINT:  Use OVERRIDING SYSTEM VALUE to override.
```
The problem here is that the insert-statements don't contain `OVERRIDING SYSTEM VALUE`.  After modifying the insert-statements to like `INSERT INTO mytable(...) OVERRIDING SYSTEM VALUE VALUES(...)` everything worked fine.
Is this an issue of ora2pg or am i missing any configuration parameters?

**Comment by darold** [comment]
Hi Robin,

This is just not implemented, I need to detect that an identity column is part of the insert and add the OVERRIDING clause. However the INSERT export method is slow you should not use it and use COPY instead. For identity columns, the COPY FROM command will always write the column values provided in the input data, like the INSERT option OVERRIDING SYSTEM VALUE. Si you will not have the problem and gain the speed :-)

I will fix that. Thanks for the report.

**Comment by R0Wi** [comment]
Thanks for your quick response. I'll try it with the `COPY`-settings and post my results here.

**Comment by darold** [comment]
Fixed in commit adafc3e
